packages/playground/src/views/settings.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+  <view-layout>
+    <v-card color="primary" class="d-flex justify-center items-center mt-3 pa-3 text-center">
+      <v-icon size="30" class="pr-3">mdi-cog</v-icon>
+      <v-card-title class="pa-0">Settings</v-card-title>
+    </v-card>
+    <v-card class="my-5"
+      ><v-card-title>Theme</v-card-title> <v-card-text>Pick an application theme!</v-card-text>
+      <form-validator v-model="isValidTheme" valid-on-init>
+        <input-validator :rules="[validateTheme]" #="{ props }" ref="themeInput" :value="selectedTheme">
+          <v-select class="pa-3" :items="themes" v-model="selectedTheme" v-bind="props" />
+        </input-validator>
+      </form-validator>
```

Comment:
Same here.

![image](https://github.com/user-attachments/assets/6005b54f-a87e-4e5c-b73a-e3ebc2226636)

Reply:
please refer to explanation above https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3297#discussion_r1715225192